org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -38,6 +38,14 @@ <h3>Fixed Bugs</h3>
       (GitHub <a href="https://github.com/jacoco/jacoco/issues/672">#672</a>).</li>
 </ul>
 
+<h3>API Changes</h3>
+<ul>
+  <li>The XML report now has an optional attribute <code>sourcefilename</code>
+      on the <code>class</code> element to allow unambiguously relate classes
+      to source files. The JaCoCo DTD version has been updated to 1.1. 
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/701">#701</a>).</li>
```

Comment:
As we discussed in https://github.com/jacoco/jacoco/pull/668#discussion_r178962005 - existing links point to PRs, not on original issue.

Reply:
Thx, fixed.